docs/gestor.md

```diff
@@ -0,0 +1,32 @@
+# Gestores de tareas y dependecias
+
+## Gestor de tareas
+
+Tras buscar los gestores de tareas que hay disponibles, para nuestro lenguaje de programación podemos seleccionar GNU make o Task.
```

Comment:
O mask, o muchos más. Sigue sin mostrar que se han examinado de verdad cuales son las mejores prácticas en el uso de task runners en Go en este momento. No se trata de que elijáis una cosa u otra, se trata de que mostréis que tenéis madurez para tomar decisiones técnicas sobre el código y el *toolchain* que vais a usar en el mismo.